### PR TITLE
ci(openbsd): skip all tests

### PR DESCRIPTION
**Pull request by dundargoc**
NOOOOOOOO you can't just comment out all the failing tests

HAHA # goes brrrrrrrrrrrrrrrrrrr

### CODE REVIEW

**Comment by dundargoc** [comment]
@jamessan I think this is the last of them :)

Lot of work ahead of us lol.

**Comment by jamessan** [comment]
Hmm, it looks like this really went south when the libuv/luv PR was merged...

**Comment by clason** [comment]
Hmmm, the problems with libuv on openBSD were supposed to be fixed...

**Comment by zeertzjq** [comment]
The libuv bump also caused a lot of functional tests failures and the oldtest hang on Windows, and now Windows CI are somehow shown as passed even though there are failed tests.

**Comment by clason** [comment]
Guess we'll have to revert that commit, then, and hope that the libluv bump doesn't require it.

**Comment by dundargoc** [comment]
We need a decision on this. I'm fine with either reverting or skipping, so long as we don't stall too much with this.

File: .builds/openbsd.yml [hunk 1]
```diff
@@ -35,9 +35,13 @@ tasks:
     cmake -G Ninja $CMAKE_EXTRA_FLAGS ..
     cmake --build . --config RelWithDebInfo
     ./bin/nvim --version
-- functionaltest: |
-    cd neovim/build
-    cmake --build . --config RelWithDebInfo --target functionaltest
```
Comment (jamessan) [inline .builds/openbsd.yml:40]: We don't actually know how much is impacted.  The tests always hang on a buffer_updates_spec.lua.  What if we instead skip that test and let the rest run?

```
env BUSTED_ARGS=--exclude-pattern=buffer_updates cmake --build . --config RelWithDebInfo --target functionaltest
```

Comment (dundargoc) [reply .builds/openbsd.yml:40]: I tried individually skipping the tests one by one until I noticed the freezes were non-deterministic, at which point I gave up. I've added your code suggestion, let's see.

Comment (dundargoc) [reply .builds/openbsd.yml:40]: Seems like there's an actual error in core/channels_spec.lua now:

```
[ RUN      ] channels can use stdio channel: 10029.50 ms ERR
test/helpers.lua:73: Expected objects to be the same.
Passed in:
(nil)
Expected:
(table: 0x0781c6982cb8) {
  [1] = 'notification'
  [2] = 'exit'
  [3] = {
    [1] = 3
    [2] = 0 } }
```

Will take a look at it tomorrow and see if I can figure out why.

**Comment by jamessan** [comment]
Reported the hang to libuv -- https://github.com/libuv/libuv/issues/3504

**Pull request closed without merging**